linux/src/threadpool_linux.c

```diff
@@ -199,7 +197,7 @@ static int threadpool_work_func(void* param)
                 }
             }
         /* Codes_SRS_THREADPOOL_LINUX_07_085: [ threadpool_work_func shall loop until threadpool_close or threadpool_destroy is called. ]*/
```

Comment:
Uhm, this is talking about close, but we changed the names to start/stop. i think we should not rename to start/stop frankly. Just makes more work....